Added 12/01 Upcoming Releases Update (#18975)
gravitational/teleport: docs/pages/preview/upcoming-releases.mdx

## Teleport

### 11.2

Teleport 11.2 will bring support for script-based automatic upgrades and

> improved Database joining experience in the Web UI.

Reviewer: ```suggestion
an improved database joining experience in the Web UI.
```

- Improved x509 integration for self-hosted Microsoft SQL Server
- ARM64 support for CentOS 7
- Signed macOS tar archives

> - Gracefully handle expired database certificates in Teleport Connect

Reviewer: ```suggestion
- Graceful handling of database certificates in Teleport Connect
```

- Fixed per-session MFA issues

> - Improved database joining flow in Web UI

Reviewer: ```suggestion
- Improved database joining flow in the Web UI
```
